tests/test.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+- name: Test Prometheus PHP FPM Exporter role
+  hosts: all
+
+  pre_tasks:
+    - name: Update apt
+      become: yes
+      apt:
+        cache_valid_time: 1800
+        update_cache: yes
+      tags:
+        - build
+
+  roles:
+    - role: sansible.prometheus_phpfpm_exporter
+
+  post_tasks:
+    - name: Get Prometheus PHP FPM Exporter service state
+      become: yes
+      service:
+        name: prometheus-phpfpm-exporter
```

Comment:
We don't have a standard for service names, he's used hyphens in the other roles so maybe we should stick to that for Prometheus.